### PR TITLE
220 - Implement error handling for API resource requesets for the samples table 

### PR DESCRIPTION
**Pull request by nozomione**
## Issue Number

**NOTE:** The **changes in 6 files** were simply the result of file renaming, import adjustments, and minor copy updates (94100446de6ab459807f6528333044c20f704611).

Closing #220

## Purpose/Implementation Notes
Implement error handling for the samples table and render an error view when error occurred.

Please see the latest UI [here](https://refinebio-web-git-nozomione-220-error-handling-for-836d3e-ccdl.vercel.app/).

## Types of changes
- Bugfix (non-breaking change which fixes an issue)

## Checklist
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/shared/SamplesTable/SamplesTableError.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import { SamplesTableEmpty } from './SamplesTableEmpty'
+
+export const SamplesTableError = ({ onClick }) => {
+  return (
+    <SamplesTableEmpty background="rgbaLight7">
+      <Box direction="row" gap="xxsmall">
+        <Text color="error">Temporarily under heavy traffic load. Please</Text>
+        <Button
+          label="try again"
+          link
```
Comment (davidsmejia) [inline src/components/shared/SamplesTable/SamplesTableError.js:12]: Is this just to refresh?

Comment (nozomione) [reply src/components/shared/SamplesTable/SamplesTableError.js:12]: This is for refetching the table data.

**Review by davidsmejia — APPROVED** [review]
LGTM

**Comment by nozomione** [comment]
🗒️ I ran the deploy scripts locally since the project counts changed for sitemap etc for Vercel preview deploys (98564de261d34c1e3e896da18d953550844160bd), thank you!

**Pull request merged**